src/neural/onnx/converter.cc

```diff
@@ -122,6 +146,28 @@ std::unique_ptr<OnnxConst> Converter::GetWeghtsConverter(
                   " is not supported in weights converter");
 }
 
+std::string Converter::MakeMish(OnnxBuilder* builder, const std::string& input,
+                                const std::string& name) {
+  auto flow = builder->Softplus(name + "/softplus", input);
+  flow = builder->Tanh(name + "/tanh", flow);
+  return builder->Mul(name, flow, input);
+}
+
+std::string Converter::MakeActivation(OnnxBuilder* builder,
+                                      const std::string& input,
+                                      const std::string& name,
+                                      ActivationFunction activation) {
+  if (activation == RELU) {
```

Comment:
Sorry for arbitrary comment, but I suspect `switch` statement would look better here than chained `if`.

Reply:
Indeed, looks a lot better.